src/Framework/Framework-uapaot.depproj

```diff
@@ -25,6 +25,7 @@
     <FileToInclude Include="mscorlib" />
 
     <!-- Pickup a few selected aot-specific files from uapaot instead -->
+    <FileToInclude Include="System.Linq" />
```

Comment:
Could you please add a comment in front of this and put it maybe in a separate block?

The non-aot version of the existing assemblies in this block are non-functional. System.Linq works fine, but we think the version optimized for size is a better fit for aot. It would be nice to keep these two groups separate.

Reply:
Yep, good call